polyfill/test/duration.mjs

```diff
@@ -525,6 +596,55 @@ describe('Duration', () => {
       equal(`${duration.subtract(Temporal.Duration.from('P1DT5M'))}`, 'P2DT1H5M');
       equal(`${duration.subtract('P1DT5M')}`, 'P2DT1H5M');
     });
+    it('relativeTo affects year length', () => {
+      const oneYear = new Duration(1);
+      const days365 = new Duration(0, 0, 0, 365);
+      equal(`${oneYear.subtract(days365, { relativeTo: Temporal.PlainDateTime.from('2017-01-01') })}`, 'PT0S');
+      equal(`${oneYear.subtract(days365, { relativeTo: Temporal.PlainDateTime.from('2016-01-01') })}`, 'P1D');
+    });
+    it('relativeTo affects month length', () => {
+      const oneMonth = new Duration(0, 1);
+      const days30 = new Duration(0, 0, 0, 30);
+      equal(`${oneMonth.subtract(days30, { relativeTo: Temporal.PlainDateTime.from('2018-02-01') })}`, '-P2D');
+      equal(`${oneMonth.subtract(days30, { relativeTo: Temporal.PlainDateTime.from('2018-03-01') })}`, 'P1D');
+      equal(`${oneMonth.subtract(days30, { relativeTo: Temporal.PlainDateTime.from('2018-04-01') })}`, 'PT0S');
+    });
+    it('first this is resolved against relativeTo, then the argument against relativeTo + this', () => {
+      const d1 = new Duration(0, 2, 1, 4);
+      const d2 = new Duration(0, 1, 1, 1);
+      const relativeTo = new Temporal.PlainDateTime(2000, 1, 1);
+      equal(`${d1.subtract(d2, { relativeTo })}`, 'P1M3D');
+    });
+    const oneDay = new Duration(0, 0, 0, 1);
+    const hours24 = new Duration(0, 0, 0, 0, 24);
+    it('relativeTo does not affect days if PlainDateTime', () => {
+      const relativeTo = Temporal.PlainDateTime.from('2017-01-01');
+      equal(`${oneDay.subtract(hours24, { relativeTo })}`, 'PT0S');
+    });
+    it.skip('relativeTo does not affect days if ZonedDateTime, and duration encompasses no DST change', () => {
+      const relativeTo = Temporal.ZonedDateTime.from('2017-01-01T00:00[America/Montevideo]');
+      equal(`${oneDay.subtract(hours24, { relativeTo })}`, 'PT0S');
+    });
+    it.skip('relativeTo does affect days if ZonedDateTime, and duration encompasses DST change', () => {
```

Comment:
Once these tests are un-skipped, I'd suggest a wider set of tests around DST transitions, e.g. 
* starts inside a repeated hour, ends in a normal hour
* starts inside a repeated hour, ends inside a skipped hour
* starts in a normal hour, ends inside a skipped hour
* starts before a skipped period, ends >1 day after
* starts before a skipped period, ends <1 day after
* starts before a repeated period, ends >1 day after
* starts before a repeated period, ends <1 day after